.gitignore

```diff
@@ -18,7 +18,7 @@ generators/**/tsconfig*tsbuildinfo
 generators/**/out
 
 # don't ignore the ruby lib within the playground
-!packages/**/ruby/playground/lib
+!generators/ruby/playground/lib
```

Comment:
This was an artifact of the move from packages to the top level generators folder that I didn't update